genfragments/ThirteenPointSixTeV/JpsiEE_Pt_2_30_13p6TeV_TuneCP5_pythia8_cfi.py

```diff
@@ -0,0 +1,53 @@
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+                         pythiaPylistVerbosity = cms.untracked.int32(0),
+                         #filterEfficiency = cms.untracked.double(0.003),
+                         pythiaHepMCVerbosity = cms.untracked.bool(False),
+                         crossSection = cms.untracked.double(1742000.0),
```

Comment:
Is this cross section correct and updated to 13.6 TeV?

Reply:
The cross-section is not updated to 13.6 TeV. There are two reasons for this. 

Firstly, I tried to look for cross-section values online and didn't find any reasonable theoretical estimate for this. In the following paper https://arxiv.org/pdf/1006.2750.pdf, the estimate is O(100 nb). Within theoretical uncertainty, there isn't an appreciable change in the production cross-section from 13 TeV to 13.6 TeV.

Secondly, the event weight can be normalized later on. 

If there is a better/more elegant/efficient solution to this, please suggest me, and I will change the value accordingly.